### ISSUE TITLE
Create notification system

### ISSUE THREAD

**Issue by corbindavenport** [ai-robots-txt/ai.robots.txt#13]
It might be useful to have an official mechanism for following updates to the repository, so site owners would know when their lists need to be updated. I just subscribed to the commits feed for the main `robots.txt` file with my RSS app:

```
https://github.com/ai-robots-txt/ai.robots.txt/commits/main/robots.txt.atom
```

That only covers changes to that file, though, I'm not sure if `ai.txt` is important here. Creating a new release for each change might be better? I'm not sure.

At minimum, it might be worth adding that RSS feed to the Readme as a basic existing notification system. I don't think many people know you can follow commits to individual files like that.

**Comment by cdransf** [comment]
👋🏻 This is a great idea! I wasn't aware that was available as a means to follow files either.

There is an atom feed available for releases, so I'll pursue that path. I've issued everything up to this point as the first and will include new bots in subsequent releases.

I added the atom link to the README and dropped it in as the primary link for the repository as well. Let me know if that sounds good and we can close this out. 🫡
